staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kmsv2/v2alpha1/types.go

```diff
@@ -21,7 +21,7 @@ package v2alpha1
 // EncryptedObject is the representation of data stored in etcd after envelope encryption.
 type EncryptedObject struct {
 	// EncryptedData is the encrypted data.
-	EncryptedData []byte `json:"data" protobuf:"bytes,2,opt,name=encryptedData"`
+	EncryptedData []byte `protobuf:"bytes,2,opt,name=encryptedData" json:"encryptedData"`
```

Comment:
I would remove the JSON tags because this should never be stored as anything but proto.